pkg/http/server.go

```diff
@@ -104,6 +104,17 @@ func (s *server) setupTLSListener(opts Opts) error {
 	}
 	config.Certificates = []tls.Certificate{cert}
 
+	if len(opts.TLS.MinVersion) > 0 {
+		switch opts.TLS.MinVersion {
+		case "TLS12", "TLS1_2", "TLS1.2", "TLS 1.2", "0x0303":
+			config.MinVersion = tls.VersionTLS12
+		case "TLS13", "TLS1_3", "TLS1.3", "TLS 1.3", "0x0304":
```

Comment:
What is the motivation for supporting many different strings here? Should we just support one format, eg TLS1.2, TLS1.3 and then document that format?

Reply:
My motivation for supporting these variants, is that based on the different supported configuration approaches, different variants would be suitable. 

In the alpha config approach with a YAML file the string `TLS 1.3` with spaces would work and look well. 
However for other config approaches, like the CLI argument, it would be weird with spaces or even dots, thus the string `TLS1_3 ` would probably be most appropriate there.

As it is easily possible to support the variants, I thought it would be great to have them.

Reply:
> As it is easily possible to support the variants, I thought it would be great to have them.

While easy to implement, it does complicate the code and extends our API, our Enum went from 2 options to 10 by supporting the different formats and I am hesitant about future possible issues that could come up trying to support all of the different variants.

My suggestion would be to make this a single option of `TLS1.3` and `TLS1.2` if that makes sense to you?

Reply:
Yes, that makes sense. I will change it to this format.

Reply:
Updated the merge request to use `TLS1.3` and `TLS1.2` only.